Source/JavaScriptCore/parser/Parser.cpp

```diff
@@ -1845,6 +1850,7 @@ template <class TreeBuilder> TreeStatement Parser<LexerType>::parseTryStatement(
             catchScope->preventVarDeclarations();
             const Identifier* ident = nullptr;
             if (matchSpecIdentifier()) {
+                semanticFailIfTrue(match(AWAIT) && !findClosetFunctionScope(), "Cannot use 'await' as identifier within class static block scope");
```

Comment:
Is it possible to use `arguments`?

Reply:
Yes, we can use `arguments` under function scope. So, these should have syntax errors:
```
    class C {
        static {
            [arguments]();
        }
    }

    class C {
        static {
            arguments[0];
        }
    }
```
But, these should work.
```
    class C {
        static {
            function inner() {
                [arguments]();
            }
        }
    }

    class C {
        static {
            function inner() {
                arguments[0];
            }
        }
    }

    class C {
        static {
            (a, b) => {
                arguments[0];
            };
        }
    }
```